src/Microsoft.Data.SqlClient/netfx/src/Microsoft.Data.SqlClient.csproj

```diff
@@ -332,14 +350,22 @@
       <EmbedInteropTypes>True</EmbedInteropTypes>
     </COMReference>
   </ItemGroup>
-  <ItemGroup />
+  <ItemGroup>
+    <Folder Include="Microsoft\SqlServer\" />
+  </ItemGroup>
   <ItemGroup>
     <PackageReference Include="Microsoft.Data.SqlClient.SNI">
-      <Version>1.1.0</Version>
+      <Version>1.0.19235.1</Version>
     </PackageReference>
     <PackageReference Include="Microsoft.Identity.Client">
       <Version>3.0.8</Version>
     </PackageReference>
+    <PackageReference Include="Microsoft.IdentityModel.JsonWebTokens">
```

Comment:
Why is this not included in NetCore and only in NetFx?

Reply:
Hm... We just test it and it looks like this is not needed for netfx any more. The `Microsoft.IdentityModel.Protocols.OpenIdConnect` will bring in this one.